e2e/es6-to-basic-javascript.spec.ts

```diff
@@ -0,0 +1,34 @@
+import test, { expect } from '@playwright/test';
+
+// To run this test locally you will need to run: pnpm run start-ci;
+// Also, make sure that you have pm2 installed globally via: pnpm install -g pm2
```

Comment:
Is this accurate?

Reply:
To my knowledge, yes, redirects don't work locally without it.

Reply:
Kinda. It's possible to serve the client without installing pm2. You just `pnpm run serve-ci` from the client directory.

It is necessary if you run everything from the root of the repo, though, since that starts the server in production(ish) mode. Yep, it's messy.